app/javascript/tests/patient/Patient.test.js

```diff
@@ -0,0 +1,186 @@
+    });
+
+    it('Properly renders address section', () => {
+        const wrapper = shallow(<Patient details={mockPatient1} groupMembers={[ ]} hideBody={true}
+            jurisdictionPath="USA, State 1, County 2" authenticity_token={authyToken} />);
+        const section = wrapper.find('#address');
+        expect(section.find(Row).first().text()).toEqual('Address');
+        expect(section.find(Button).length).toEqual(0);
+        expect(section.find('.font-weight-light').at(0).text()).toEqual('1 Hartford Drive');
+        expect(section.find('.font-weight-light').at(1).text()).toEqual('Springfield Connecticut Fairfield 00000-0000');
+    });
+
+    it('Properly renders arrival information section', () => {
+        const wrapper = shallow(<Patient details={mockPatient1} groupMembers={[ ]} hideBody={true}
+            jurisdictionPath="USA, State 1, County 2" authenticity_token={authyToken} />);
+        const section = wrapper.find('#arrival-information');
+        expect(section.find(Row).first().text()).toEqual('Arrival Information');
+        expect(section.find(Button).length).toEqual(0);
+        expect(section.find('h6').at(0).text()).toEqual('DEPARTED');
+        expect(section.find('.font-weight-light').at(0).text()).toEqual('Cabo');
+        expect(section.find('.font-weight-light').at(1).text()).toEqual('09/08/2020');
+        expect(section.find('h6').at(1).text()).toEqual('ARRIVAL');
+        expect(section.find('.font-weight-light').at(2).text()).toEqual('Orlando');
+        expect(section.find('.font-weight-light').at(3).text()).toEqual('09/10/2020');
+        expect(section.find('.font-weight-light').at(4).text()).toEqual('Spirit');
+        expect(section.find('.font-weight-light').at(5).text()).toEqual('1515');
+    });
+
+    it('Properly renders additional planned travel section', () => {
+        const wrapper = shallow(<Patient details={mockPatient1} groupMembers={[ ]} hideBody={true}
+            jurisdictionPath="USA, State 1, County 2" authenticity_token={authyToken} />);
+        const section = wrapper.find('#additional-planned-travel');
+        expect(section.find(Row).first().text()).toEqual('Additional Planned Travel');
+        expect(section.find(Button).length).toEqual(0);
+        additionalTravelFields.forEach(function(field, index) {
+            expect(section.find('.font-weight-normal').at(index).text()).toEqual(field+':');
+        });
+    });
+
+    it('Properly renders case information section', () => {
+        const wrapper = shallow(<Patient details={mockPatient1} groupMembers={[ ]} hideBody={true}
+            jurisdictionPath="USA, State 1, County 2" authenticity_token={authyToken} />);
+        const section = wrapper.find('#exposure-case-information');
+        expect(section.find(Row).first().text()).toEqual('Case Information');
+        expect(section.find(Button).length).toEqual(0);
+        expect(section.find('.font-weight-light').at(0).text()).toEqual('Symptom Onset: 09/27/2020');
```

Comment:
I would make sure there are unique tests for testing that this field is shown when in isolation workflow, and not shown when in exposure workflow. Same with all the other fields that are specific to a workflow - want to make sure you test both cases.

Reply:
I test both exposure and isolation in this case (this is the only section in the file that shows different things depending on the workflow.  See the test before this for the one for what is displayed if the monitoree is in the isolation workflow (also copied them below).

```
it('Properly renders case information section', () => {
        const wrapper = shallow(<Patient details={mockPatient1} groupMembers={[ ]} hideBody={true}
            jurisdictionPath="USA, State 1, County 2" authenticity_token={authyToken} />);
        const section = wrapper.find('#exposure-case-information');
        expect(section.find(Row).first().text()).toEqual('Case Information');
        expect(section.find(Button).length).toEqual(0);
        expect(section.find('.font-weight-light').at(0).text()).toEqual('Symptom Onset: ' + dateFormatter(mockPatient1.symptom_onset));
        expect(section.find('.font-weight-light').at(1).text()).toEqual('Case Status: ' + mockPatient1.case_status);

    });

    it('Properly renders potential exposure information section', () => {
        const wrapper = shallow(<Patient details={mockPatient2} groupMembers={[ ]} hideBody={true}
            jurisdictionPath="USA, State 1, County 2" authenticity_token={authyToken} />);
        const section = wrapper.find('#exposure-case-information');
        expect(section.find(Row).first().text()).toEqual('Potential Exposure Information');
        expect(section.find(Button).length).toEqual(0);
        expect(section.find('h6').text()).toEqual('LAST EXPOSURE');
        expect(section.find('.font-weight-light').at(0).text()).toEqual(`${mockPatient2.potential_exposure_location} ${mockPatient2.potential_exposure_country}`);
        expect(section.find('.font-weight-light').at(1).text()).toEqual(dateFormatter(mockPatient2.last_date_of_exposure));
        potentialExposureFields.forEach(function(field, index) {
            expect(section.find('.text-danger').at(index).text().includes(field)).toBeTruthy();
        });
    });
```